doc/source/install/others.rst

```diff
@@ -242,6 +242,78 @@ files::
 
 Shows all ``configure`` options.
 
+``--enable-apache-arrow``
++++++++++++++++++++++++++
+
+Enables Apache Arrow features.
+
+The default is ``auto``. The features are enabled if Apache Arrow 
+is installed on the system, otherwise disabled.
+
+When you specify this parameter expressly, you should install Apache
+Arrow on the system beforehand.
+
+You can install Apache Arrow according to official installation procedure.
+Groonga requires either ``libarrow-dev`` or ``arrow-devel`` depending on the system.
+
+`Install Apache Arrow <https://arrow.apache.org/install/>`_
+
+Here is an example that enables Apache Arrow features expressly::
+
+  % ./configure --enable-apache-arrow
+
+.. note::
+
+   If you install Apache Arrow manually, use the :ref:`install-others-configure-pkg-config-path` option.
+
+``--disable-apache-arrow``
+++++++++++++++++++++++++++
+
+Disables Apache Arrow features.
+
+The default is ``false``.
+
+When this parameter is specified, Groonga disables Apache Arrow 
+features even if Apache Arrow is install on the system.
+
+Here is an example that disables Apache Arrow features::
+
+  % ./configure --disable-apache-arrow
+
+.. _install-others-configure-pkg-config-path:
+
+``PKG_CONFIG_PATH=PATHS``
++++++++++++++++++++++++++
+
+Adds search paths of ``.pc`` files for ``pkg-config``.
+
+Groonga uses ``pkg-config`` to use external libraries.
+
+``pkg-config`` is a tool that sets flags and paths required for using
+external libraries by simply preparing ``.pc`` files.
+
+When using libraries installed via a package manager, you do not need
+to specify this parameter because ``.pc`` files for the libraries are in
+the default search paths of ``pkg-config``.
+
+When you use manual built libraries, you need to specify ``.pc`` file paths 
+of the libraries for this parameter.
```

Comment:
```suggestion
of these libraries by this parameter.
```

Reply:
I feel like `those` or `the` is more suitable for this sentence than `these`.

Reply:
so I will fix this sentence to `those libraries by this parameter.`.